e2e/playwright/tests/folder-rules/fixtures/page-initialization.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2005 - 2023 Alfresco Software, Ltd. All rights reserved.
+ *
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
+ */
+
+import { PersonalFilesPage } from '../page-objects';
+import { test as base } from '@playwright/test';
+import { ApiClientFactory } from '../../../shared/api/api-client-factory';
```

Comment:
should probably be `@alfresco/playwright-shared`

Reply:
Thanks. I should've probably marked the PR as `not ready to review` as I only created it to debug travis builds :)